btcjson/helpers.go

```diff
@@ -75,3 +75,11 @@ func String(v string) *string {
 	*p = v
 	return p
 }
+
+// NewFilterTypeName is a helper routine that allocates a new FilterTypeName value to store v and
+// returns a pointer to it.  This is useful when assigning optional parameters.
+func NewFilterTypeName(v FilterTypeName) *FilterTypeName {
+	p := new(FilterTypeName)
+	*p = v
+	return p
+}
```

Comment:
Was about to comment that it might make sense to make the `const FilterTypeBasic` a `var` so it would be possible to do `&FilterTypeBasic`, but seems like you already anticipated this and added a nice solution:-)

Reply:
Wasn't super happy about adding helper for this but it allows using `const` instead of `var` and is pretty consistent with how `rpcclient` API is used